django/db/migrations/migration.py

```diff
@@ -219,6 +219,14 @@ def suggest_name(self):
             name = new_name
         return name
 
+    def check(self):
+        errors = []
+        for operation in self.operations:
+            operation_errors = operation.check()
+            if hasattr(operation_errors, "__iter__"):
+                errors.extend(operation.check())
```

Comment:
IMO this check is unnecessary:
```suggestion
            errors.extend(operation.check())
```

Reply:
Didn't find anything better 🤔 
Without this condition, unit tests are failing with a `Mock() is not iterable` because of the Migration test class in `tests/postgres_tests/migrations/0001_setup_extensions.py`:
```
class Migration(migrations.Migration):

    operations = [
        ...
        # CryptoExtension is required for RandomUUID() on PostgreSQL < 13.
        CryptoExtension() if needs_crypto_extension else mock.Mock(), <--- create a Mock operation
        ...
    ]
```

Another option could be to implement a `NoopOperation` class for unit tests maybe? What do you think? :)

Reply:
alternative, create a "MigrationMock" factory 

```py
def MigrationMock():
     m = Mock()
     m.check.return_value = []
     return m

# usage
m = MigrationMock()

assert len(m.check()) == 0

assert m.check.called
```